### ISSUE TITLE
How to draw a Bezier in animation?

### ISSUE THREAD

**Issue by microkof** [caleb531/jcanvas#365]
I want to draw a Bezier in 1s from right bottom to top left.

init:

```
$('#banner-canvas').drawBezier({
      layer: true,
      name: 'LeftBezier',
      strokeStyle: '#1386f1',
      strokeWidth: 2,
      x1: 470,
      y1: 70,
      x2: 470,
      y2: 70,
    });
```

animate:

```
$('#banner-canvas').animateLayer(
            'LeftBezier',
            {
              x1: 470,
              y1: 70,
              cx1: 420,
              cy1: 70,
              cx2: 420,
              cy2: 10,
              x2: 370,
              y2: 10,
            },
            1000,
            'linear',
          );
```

But the animation process is not what I want, because cx1/cy1/cx2/cy2 are also involved in animation.

How to write the code?

Thank you!

**Comment by caleb531** [comment]
@microkof That will probably be very challenging to implement, because all of the points would need to animate on the path of your Bezier _as you're drawing the curve_. However, jCanvas (and jQuery) animations can only animate directly from point A to point B (i.e. in a straight line, in this case). Changing the easing won't help, because easing only affects the timing, not direction/position.

In other words, I'm sorry to say I don't think it's possible with jCanvas.
